### PR TITLE
(#3747) - Add edit button to content

### PR DESCRIPTION
**Pull request by nickcolley**
Adds a 'Edit this on Github' button to relevant page.

This opens up the Github inspector ready to edit.

### CODE REVIEW

**Comment by nickcolley** [comment]
Looks like this:

image

**Comment by nolanlawson** [comment]
This is fantastic, but I find it a bit distracting in the guides and the API:

screenshot 2015-06-28 13 31 46
screenshot 2015-06-28 13 32 28

I feel like maybe the text should be shorter and/or it should maybe look like a button to set it apart somehow. I think the GWT project does this really well, with a nice small icon and some hover-over text: http://www.gwtproject.org/doc/latest/DevGuide.html

If it were shorter we could also maybe make it float right so that it doesn't take up its own line (which looks kind of odd in the API part to me).

**Comment by nickcolley** [comment]
Great points I'll see if I can address them.

**Comment by nickcolley** [comment]
OK I've changed it so we now have them next to the titles and anchors.

More subtle, let me know what you think.

image
image

**Comment by daleharvey** [comment]
Will give a test run in the morning, but I seriously love this, could make it hugely easier to contribute to the docs

**Comment by daleharvey** [comment]
heh sorry

**Comment by daleharvey** [comment]
There is a broken link from API Overview to https://github.com/pouchdb/pouchdb/edit/master/docs/_includes/api/overview.html

File: docs/api.html [hunk 1]
```diff
@@ -2,11 +2,12 @@
 layout: 2ColLeft
 title: API Reference
 sidebar: api.html
+edit: false
 ---
 
 {% markdown %}
 
-{% include api/api_overview.html %}
+{% include api/overview.html %}
```
Comment (daleharvey) [inline docs/api.html]: Not quite sure whats happening here but something broke, I see https://github.com/pouchdb/pouchdb/blob/master/docs/_includes/api/api_overview.html in the repo, but overview.html locally

**Comment by daleharvey** [comment]
Ok, one minor issue with a broken link, otherwise very happy to get this merged

File: docs/api.html (hunk 1 above)
Comment (nickcolley) [inline docs/api.html]: The anchor tag is #overview so I changed the file name entirely.

Comment (daleharvey) [inline docs/api.html]: ah ok

**Comment by daleharvey** [comment]
Awesome, my bad for not noticing the rename

Merged in https://github.com/pouchdb/pouchdb/commit/790c5e9e6008b53fcf44f9f32e9ba0032f9a89a8

I think we should do a new release on the 1st and publish the site around the same time, seriously excited to be using this, I forgo a lot of minor documentation edits because of the process, will be doing a lot via this I think

**Comment by nolanlawson** [comment]
@nickcolley Looks incredible! Nice work.

@daleharvey I'm down for a release on the 1st. I did 3.6.0; your turn this time? :)

**Pull request closed without merging**